randovania/games/factorio/gui/ui_files/games_tab_factorio_widget.ui

```diff
@@ -107,7 +112,100 @@
            <string># updated from code</string>
           </property>
           <property name="alignment">
-           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
+           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
+          </property>
+          <property name="wordWrap">
+           <bool>true</bool>
+          </property>
+         </widget>
+        </item>
+       </layout>
+      </widget>
+     </widget>
+    </item>
+   </layout>
+  </widget>
+  <widget class="QWidget" name="differences_tab">
+   <attribute name="title">
+    <string>Differences</string>
+   </attribute>
+   <layout class="QVBoxLayout" name="differences_layout">
+    <property name="leftMargin">
+     <number>0</number>
+    </property>
+    <property name="topMargin">
+     <number>0</number>
+    </property>
+    <property name="rightMargin">
+     <number>0</number>
+    </property>
+    <property name="bottomMargin">
+     <number>0</number>
+    </property>
+    <item>
+     <widget class="QScrollArea" name="differences_scroll_area">
+      <property name="widgetResizable">
+       <bool>true</bool>
+      </property>
+      <widget class="QWidget" name="differences_scroll_area_contents">
+       <property name="geometry">
+        <rect>
+         <x>0</x>
+         <y>0</y>
+         <width>489</width>
+         <height>615</height>
+        </rect>
+       </property>
+       <layout class="QVBoxLayout" name="differences_scroll_layout">
+        <item>
+         <widget class="QLabel" name="differences_label">
+          <property name="text">
+           <string>### Freebies
+Whenever a technology is researched, any eligible item from the unlocked recipes are distributed to online players as bonus.
+
+Using buildings adquired by freebies is expected by logic, such as solar panels for power.
+
+#### Eligible Items
+- Placeable entities, such as buildings and robots.
+- Armors, equipment and guns.
+
+### New Items and Recipes
+- Burner Assembling Machine and Burner Lab have been added, always available.
+- Longer Handed Inserter have been added with it's own technology.
+- Alternative recipes for handcrafting Construction and Logistic Robots were added.
+
+### Starting Items and Recipes
+- Transport Belts are now unlocked by Logistics 1, along with Underground Belts and Splitters.
+- Pipes and Pipe to Ground are now unlocked by default, instead of Steam Power.
+- Copper Cable, Small Electric Poles, Iron Stick and Labs are once again unlocked by default.
+
+### Inserter Changes
+- Inserter have it's own technology.
```

Comment:
Inserters now have their own technology, progressive with Fast Inserters and Bulk Inserters.

Reply:
I actually wrote about being progressive at first, then decided to skip mentioning these. Not opposed, was just trying to reduce the number of words